.github/CONTRIBUTING.md

```diff
@@ -95,3 +95,17 @@ time you run `make build`, you can symlink the built binary into the directory:
 ln -s $GOPATH/bin/terraform-provider-google ~/.terraform.d/plugins/terraform-provider-google
 ln -s $GOPATH/bin/terraform-provider-google-beta ~/.terraform.d/plugins/terraform-provider-google-beta
 ```
+
+# Maintainer-specific information
+
+## Reviewing / Merging Code
+
+When reviewing code, follow the guidelines set in the
+[Maintainer's Etiquette](https://github.com/hashicorp/terraform/blob/master/docs/maintainer-etiquette.md)
+guide. If reviewing another maintainer, it's expected that they'll merge their own code. Otherwise, merge at will once
```

Comment:
This is a repeat of a more detailed version in the maintainer's etiquette and this concise version doesn't make things more clear. I would just defer to the linked version.

Reply:
Huh, I hadn't clicked through to that in a while. I guess our teamsite duplicated this a whole bunch too. Added a caveat that the maintainer guide is pretty old (it was from the pre-provider split days) but that general guidance should be followed.